### ISSUE TITLE
Metadata import

### ISSUE THREAD

**Issue by isedwards** [climsoft/climsoftweb#16]
Unless the software is hosted internally within an organisation, Climsoft Web does not have a direct connection to the orgnisations master Climsoft Destktop database.

Therefore it is not intended that metadata should be editable in Climsoft Web because the data would be maintained separately in two different places.

There should be an option to regularly import the latest version of the metadata from the master database. This needs to be a managed process: If a station or element previously existed and a full metadata update would remove the station/element then the user must be made aware of the consequences for existing key entry forms (do we need a "dry run" option).

This business logic must also be captured for the future Data APIs.

**Comment by mhabimana** [comment]
@isedwards :Sounds good. There is a need to specify as requirements that need to be in place to host Climsoft web internally in a given Met Service in terms of Hardware,software,network,etc. rather than in a cloud or external hosting place so that preparations can be put in place.

**Comment by isedwards** [comment]
Agreed, although it's not clear at the moment that securely hosting a web application is an option for a small Met Service.

I think the first prerequisite is that they must already be hosting their organisation's web site from a web server within their building.  The web sites of some Met Services run from an external company or data center and not from a machine within the organisation.

If a service is already hosting their web presence on site, then this means that they will already have:
- A suitable server room with a dedicated internet connection
- Public IP address(es)
- Firewall
- IT staff who can manage the network securely including regular security updates
- Back up UPS (power supply in case of power loss where internet connectivity remains)
-  ...
